image_proc/src/resize.cpp

```diff
@@ -54,16 +54,19 @@ ResizeNode::ResizeNode(const rclcpp::NodeOptions & options)
     {
       auto result = rcl_interfaces::msg::SetParametersResult();
       result.successful = true;
+      image_topic_ = "/camera";
```

Comment:
You shouldn't hardcode the `/` since its going to be gotten by a parameter. Parameters convention is not to include `/`.